test/unit/test-element-service.js

```diff
@@ -38,30 +38,11 @@ import {
 describe('getElementServiceIfAvailable()', () => {
   let doc;
   let win;
-  let readyStateListeners;
```

Comment:
I'm assuming this is just a revert of the original PR?

Reply:
Pretty much, yes.